Formula/percona-toolkit.rb

```diff
@@ -11,8 +11,8 @@ class PerconaToolkit < Formula
   head "lp:percona-toolkit", using: :bzr
 
   livecheck do
-    url "https://www.percona.com/downloads/percona-toolkit/LATEST/"
-    regex(%r{value=.*?percona-toolkit/v?(\d+(?:\.\d+)+)["' >]}i)
+    url "https://docs.percona.com/percona-toolkit/version.html"
+    regex(/Percona Toolkit v?(\d+(?:\.\d+)+) released/i)
```

Comment:
```suggestion
    regex(/Percona\s+Toolkit\s+v?(\d+(?:\.\d+)+)\s+released/im)
```

* When checking loose text, I generally use `\s+` to match whitespace that we expect to exist (and `\s*` otherwise) instead of a literal space. There have been instances in the past where a check has broken simply because there was more than one space or a slightly different type of space character was used, etc. and `\s+` handles some of that (though not something like ` &nbsp;`).
* I also try to make sure to use the multiline (`m`) regex option when checking loose text, as I've seen regexes fail to match because a line break was inserted somewhere in the text in an update. This goes hand in hand with `\s+`, as it also covers newline characters.